src/libs/ReportUtils.ts

```diff
@@ -459,6 +459,7 @@ type OptionData = {
     tabIndex?: 0 | -1;
     isConciergeChat?: boolean;
     isBold?: boolean;
+    lastIOUTime?: string;
```

Comment:
```suggestion
    lastIOUCreationDate?: string;
```

Just trying to see if we can make this a bit more descriptive